lib/DDGC/Schema/Result/User.pm

```diff
@@ -4,6 +4,8 @@ package DDGC::Schema::Result::User;
 
 use Moo;
 extends 'DDGC::Schema::Result';
+with 'DDGC::Schema::Role::Result::User::Subscription';
+
```

Comment:
I genuinely prefer reading the multi-line column defs - I think I get bored if my eyes have to stray to far to the right :wink: ...though if there are dozens of them, that is a sign that something else may be amiss. Still, rather that than these damned JSON / YAML serialised columns we have. I did spot your single line column defs in the Github work, it does *look* very tidy.

I can see these Result classes become column defs and a bunch of `with` specifications, but there comes a point when too many files is as rough to deal with as long files. I hope I can keep my scatterbrain on an appropriately coarse granularity.

Thanks for input!